src/SampleWebApp/Controllers/AuthorssApiController.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/AuthorssApi")]
+    public class AuthorssApiController : Controller
+    {
+        private AuthorDataContext _context;
+
+        public AuthorssApiController(AuthorDataContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/AuthorssApi
+        [HttpGet]
+        public IEnumerable<Author> GetAuthor()
+        {
+            return _context.Author;
+        }
+
+        // GET: api/AuthorssApi/5
+        [HttpGet("{id}", Name = "GetAuthor")]
+        public IActionResult GetAuthor([FromRoute] int id)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            Author author = _context.Author.SingleOrDefault(m => m.AuthorID == id);
```

Comment:
Was the `.Author` property generated as part of this scaffold? Is that preferable to `Authors`?